@xen-orchestra/web-core/lib/components/table/UiTable.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <table :class="{ 'vertical-border': verticalBorder, error: color === 'error' }" class="ui-table typo p2-regular">
+    <slot />
+  </table>
+</template>
+
+<script lang="ts" setup>
+import { provide } from 'vue'
+
+const props = defineProps<{
+  color?: 'error'
+  name?: string
+  verticalBorder?: boolean
+}>()
+
+provide('tableName', props.name)
+</script>
+
+<style lang="postcss" scoped>
+.ui-table {
```

Comment:
Based on the comment for the colors in `ColumnHeader.vue`, you can set the background color like this:
```suggestion
.ui-table {
  --background-color: var(--background-color-primary);

  &.error {
    --background-color: var(--background-color-red-10);
  }
}
```